docs/BEMAN_PROCESS.md

```diff
@@ -0,0 +1,48 @@
+<!--
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+
+# The Beman Process
+
+This document specifies stages and best practices for Beman Project libraries development. Its goal is to create consistency facilitating the evaluation of, and contribution to Beman libraries.
+
+## Introduction
+
+### Changing this document
+
+This is a living document that adapts to evolving best practices and community
+needs. To make changes:
+
+1. Create a [discourse topic](https://discourse.boost.org) detailing the change
```

Comment:
Should it be the https://discourse.bemanproject.org/?

Reply:
Nice catch! I just copied it from BEMAN_STANDARD. I will update in both places.